assets/src/application/ui/layout/EspressoDropdownMenu/style.scss

```diff
@@ -0,0 +1,489 @@
+		&:hover,
+		&:not(:disabled):not([aria-disabled='true']):not(.is-secondary):not(.is-tertiary):not(.is-primary):hover {
+			color: #191e23;
+			border: none;
+			background: #f3f4f5 !important;
+			box-shadow: none;
+		}
+	}
+
+	.components-menu-item {
+		&:hover,
+		&:not(:disabled):not([aria-disabled='true']):not(.is-secondary):not(.is-tertiary):not(.is-primary):hover {
+			color: #191e23;
+			border: none;
+			background: #f3f4f5 !important;
+			box-shadow: none;
+		}
+	}
+
+	.components-dropdown-menu__menu-item > svg,
+	.components-menu-item > svg {
+		border-radius: 4px;
+		padding: 2px;
+		width: 24px;
+		height: 24px;
+		margin: -1px 8px -1px 0;
+	}
+
+	.components-dropdown-menu__menu-item:not(:disabled):not([aria-disabled='true']):not(.is-secondary).is-active > svg,
+	.components-menu-item:not(:disabled):not([aria-disabled='true']):not(.is-secondary).is-active > svg {
+		outline: none;
+		color: #fff;
+		box-shadow: none;
+		background: #555d66;
+	}
+
+	.components-menu-group:not(:last-child) {
+		border-bottom: 1px solid #e2e4e7;
+	}
+
+	.components-menu-item__button {
+		height: auto;
+		padding-left: 2rem;
+
+		&.components-button {
+			height: auto;
+			padding-left: 2rem;
+
+			&.has-icon {
+				padding-left: 0.5rem;
+			}
+		}
+
+		&.has-icon {
+			padding-left: 0.5rem;
+		}
+
+		&.components-button .dashicon,
+		.dashicon {
+			margin-right: 4px;
+		}
+	}
+
+	.components-dropdown-menu__menu-item:focus:not(:disabled):not([aria-disabled='true']):not(.is-secondary),
+	.components-menu-item:focus:not(:disabled):not([aria-disabled='true']):not(.is-secondary) {
+		color: #191e23;
+		border: none;
+		box-shadow: none;
+		outline-offset: -2px;
+		outline: 1px dotted #555d66;
+	}
+
+	.components-dropdown-menu__menu-item {
+		&:hover,
+		&:not(:disabled):not([aria-disabled='true']):not(.is-secondary):not(.is-tertiary):not(.is-primary):hover {
+			color: #191e23;
+			border: none;
+			background: #f3f4f5;
+			box-shadow: none;
+		}
+	}
+
+	.components-menu-item {
+		&:hover,
+		&:not(:disabled):not([aria-disabled='true']):not(.is-secondary):not(.is-tertiary):not(.is-primary):hover {
+			color: #191e23;
+			border: none;
+			background: #f3f4f5;
+			box-shadow: none;
+		}
+	}
+
+	.components-dropdown-menu__menu-item > svg,
+	.components-menu-item > svg {
+		border-radius: 4px;
+		padding: 2px;
+		width: 24px;
+		height: 24px;
+		margin: -1px 8px -1px 0;
+	}
+
+	.components-dropdown-menu__menu-item:not(:disabled):not([aria-disabled='true']):not(.is-secondary).is-active > svg,
+	.components-menu-item:not(:disabled):not([aria-disabled='true']):not(.is-secondary).is-active > svg {
+		outline: none;
+		color: #fff;
+		box-shadow: none;
+		background: #555d66;
+	}
+
+	.components-menu-group:not(:last-child) {
+		border-bottom: 1px solid #e2e4e7;
+	}
+
+	.components-menu-item__button {
+		height: auto;
+		padding-left: 2rem;
+
+		&.components-button {
+			height: auto;
+			padding-left: 2rem;
+		}
+	}
+}
+
+.components-popover {
```

Comment:
For some reasons, styles from popover were not automatically imported, so I had to add these manually.